javascript/modules/client/src/internal/interfaces/client.ts

```diff
@@ -82,6 +87,54 @@ export interface IWithdrawParams {
   tokenAddress?: string;
   reference?: string;
 }
+export interface IPermissionParams {
+  where: string;
+  who: string;
+  permission: DaoPermissions;
+}
+export interface IFreezeParams {
+  where: string;
+  permission: DaoPermissions;
+}
+
+export enum DaoPermissions {
```

Comment:
Could we create an object with the permissions computed, instead of hardcoded?

If malicious mutability of values is a concern, we can export a Class with static getters:
```ts
export class DaoPermissions {
  get UPGRADE_PERMISSION_ID() {
    return keccak256("...");
  }
  ...
}
````

Reply:
Yes, this will imply that we will need to make the permission of type string, the return of the getter, which forces us to validate that the permission is in a list of permissions which implies having a map between permissions and permission hash, which is what we were trying to avoid with the class